sealink/ticket_printer
# Migrate from Travis to GitHub Action

**WHY**

- Upgrade Gradle from 3.4 to 6.7
- Migrate to Github Action from Travis CI.
- Migrate to Github Package from Bintray.

**How to test:**
- Check the workflow in the [Actions](https://github.com/sealink/ticket_printer/actions).
- Check the package generated in [Github Packages list](https://github.com/mark-onenetwork/ticket_printer/packages/961688)
